src/App.js

```diff
@@ -1,9 +1,91 @@
 import React from 'react';
 import './App.css';
+import Button from './assets/components/Button.js';
+import Product from './assets/components/Product';
+import Tile from './assets/components/Tile';
+import bag1 from './assets/bag_1.png'
+import bag2 from './assets/bag_2.png'
+import bag3 from './assets/bag_3.png'
+import bag4 from './assets/bag_4.png'
+import brand from './assets/brand.png'
+import story from './assets/our_story.png'
+
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
+      <>
+            <h1>Handbags & Purses</h1>
+        <nav>
+        <Button
+        type= "button"
+        click= {() => console.log ("to the collection")}
```

Comment:
ander manier had kunnen zijn:
        <Button isDisabled={false} clickHandler={() => console.log("To the collection!")}>
          To the collection
        </Button>
        <Button isDisabled={false} clickHandler={() => console.log("Shop all bags!")}>
          Shop all bags
        </Button>
        <Button isDisabled={true} clickHandler={() => console.log("To the pre-orders!")}>
          Pre-orders
        </Button>